tcms/rpc/api/forms/__init__.py

```diff
@@ -35,6 +35,10 @@ def __init__(  # pylint: disable=too-many-arguments
         instance=None,
         use_required_attribute=None,
         renderer=None,
+        start_date=None,
+        stop_date=None,
+        planned_start=None,
+        planned_stop=None,
```

Comment:
These changes do not belong here. They don't belong anywhere. 

The purpose of UpdateModelFormMixin is to make all fields not required so that API calls can update them one by one. This mixin class is used for all model forms and it's parameters are unrelated to the model on which it operates.  See `ModelForm.__init__()` implementation in Django's source code for more details.

Reply:
so this basically means that the TestRun.update() will not have the parameters required, right?

Reply:
It means that any model form which inherits from `UpdateModelFormMixin` will automatically reset the `required` attribute for all of its fields regardless of how they are declared in the source code. This applies to `UpdateForm` as well.

If you inspect the changes from your previous PR you will see that you have added new fields to UpdateForm and none of them are listed as `required=False`, e.g. all of them are defined as `required=True` implicitly. The init method of the mixin class modifies this behavior.

Reply:
got it thanks!